Mathlib/Order/ConditionallyCompleteLattice/Finset.lean

```diff
@@ -75,6 +75,16 @@ theorem Set.Finite.lt_cInf_iff (hs : s.Finite) (h : s.Nonempty) : a < sInf s ↔
 
 end ConditionallyCompleteLinearOrder
 
+section ConditionallyCompleteLinearOrderBot
+variable [ConditionallyCompleteLinearOrderBot α]
+
+lemma Finset.sup_univ_eq_ciSup [Fintype ι] (f : ι → α) : univ.sup f = ⨆ i, f i :=
+  le_antisymm
+    (Finset.sup_le fun _ _ => le_ciSup (finite_range _).bddAbove _)
+    (ciSup_le' fun _ => Finset.le_sup (mem_univ _))
```

Comment:
@YaelDillies, perhaps you know if this is hiding elsewhere

Reply:
No, this is indeed missing. Note however that it belongs in the next section

Reply:
How so?

Reply:
I span off #9807; the structure of this file was confusing me because it had the same lemma twice in different places.